# the time stamp of getting this data in the ROS system
time stamp

# the time stamp in FC
uint32 fc_timestamp_us
uint32 fc_utc_yymmdd
uint32 fc_utc_hhmmss
